### PR TITLE
Add Qeb-Hwt workflow

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Add Qeb-Hwt GitHub App integration of Thoth adviser.

### CODE REVIEW

File: adviser/base/argo-workflows/qeb-hwt-integration.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          - name: GITHUB_HEAD_REPO_URL
+            value: "{{inputs.parameters.GITHUB_HEAD_REPO_URL}}"
+          - name: ORIGIN
+            value: "{{inputs.parameters.ORIGIN}}"
+          - name: THOTH_HOST
+            value: "{{inputs.parameters.THOTH_HOST}}"
+          - name: REPO_PATH
+            value: "{{inputs.parameters.REPO_PATH}}"
+          - name: "WEBHOOK_CALLBACK_URL"
+            value: "http://qeb-hwt-aicoe-prod-bots.cloud.paas.psi.redhat.com"
```
Comment (goern) [inline adviser/base/argo-workflows/qeb-hwt-integration.yaml:58]: can we have a generic value for this? or, we need to put this configuration into a configmap that is created via an overlay. the webhook callback URL seems to be environment specific, therefor the configmap seems to be a good way.

File: adviser/base/argo-workflows/qeb-hwt-integration.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: thamos-advise
+  annotations:
```
Comment (goern) [inline adviser/base/argo-workflows/qeb-hwt-integration.yaml:6]: annotations and labels should be removed, they are managed via the kustomization

File: adviser/base/argo-workflows/qeb-hwt-integration.yaml (hunk 1 above)
Comment (pacospace) [reply adviser/base/argo-workflows/qeb-hwt-integration.yaml:58]: but this is a prod value? do we have a test env for the github-webhook receiver? In that case I will create the value in core/base/configmaps.yaml in test? @goern

File: core/base/configmaps.yaml [hunk 3]
```diff
@@ -36,3 +36,11 @@ kind: ConfigMap
 apiVersion: v1
 metadata:
   name: prometheus
+---
```
Comment (goern) [inline core/base/configmaps.yaml:39]: Let’s keep application specific config with the application,

File: adviser/base/openshift-templates/qeb-hwt.yaml [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  annotations:
+    description: "Thoth: qeb-hwt Template"
+    openshift.io/display-name: "Thoth: qeb-hwt"
+    version: 0.2.0
+    tags: thoth,ai-stacks,aistacks,qeb-hwt,machinelearning
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to run thamos advise workflow for qeb-hwt Template.
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+  labels:
```
Comment (goern) [inline adviser/base/openshift-templates/qeb-hwt.yaml:14]: Could be removed

**Pull request merged**